PRODDOCS-670: Lint multiple files (#8611)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/studio-ide/lint-format.md

Reviewer: ```suggestion
- Works with Jinja and SQL.
```

<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-lint-format-console.gif" width="90%" title="Use the Lint or Fix button in the console section to lint or auto-fix your code."/>

### Lint multiple files

You can lint multiple SQL files at once, depending on how you are working with dbt. The behavior differs between the <Constant name="cloud_ide" />, <Constant name="core" />, and the <Constant name="cloud_cli" />.

> - **<Constant name="cloud_ide" />** &mdash; Linting in the <Constant name="cloud_ide" /> runs against [all modified](/docs/cloud/studio-ide/lint-format#snapshot-linting) `.sql files` in your project by default on your current branch.

Reviewer: ```suggestion
- **<Constant name="cloud_ide" />**: By default, linting runs against all modified `.sql` files in your project on your current branch. See [Snapshot linting](#snapshot-linting) for more information.
```

> - **<Constant name="core" />** &mdash; <Constant name="core" /> does not include a built-in linter. To lint SQL files in your project, use a third-party tool such as SQLFluff with the dbt templater. You can lint multiple files by specifying one or more files or directory `PATHS` flags.

Reviewer: Should we add this link https://docs.sqlfluff.com/en/stable/configuration/templating/dbt.html to provide info about the dbt templater? 
```suggestion
- **<Constant name="core" />**: Does not include a built-in linter. To lint SQL files in your project, use a third-party linter such as SQLFluff configured to use the dbt templater. You can lint multiple files by specifying one or more files or directory `PATHS` flags.
```
Also, the last sentence is a bit unclear. Do you mean "You can lint multiple files by specifying one or more file paths or directory paths as arguments to the command."?

> - **<Constant name="core" />:** Does not include a built-in linter. To lint SQL files in your project, use a third-party linter such as SQLFluff configured to use the [dbt templater](https://docs.sqlfluff.com/en/stable/configuration/templating/dbt.html). You can lint multiple files by specifying one or more files or directory `PATHS` flags.

Author: ```suggestion
- **<Constant name="core" />:** Does not include a built-in linter. To lint SQL files in your project, use a third-party linter such as SQLFluff configured to use the [dbt templater](https://docs.sqlfluff.com/en/stable/configuration/templating/dbt.html). You can lint multiple files by specifying one or more file or directory paths as arguments to the command.
```

> - **<Constant name="cloud_cli" />** &mdash; The <Constant name="cloud_cli" /> supports the same linting [commands](/reference/dbt-commands) as dbt Core:

Reviewer: ```suggestion
- **<Constant name="cloud_cli" />**: Supports the same linting [commands](/reference/dbt-commands) as dbt Core:
```
The page linked here does not have any info about linting. Should we use the link in line 85 instead?

> 

Reviewer: If lines 79-85 should be under dbt CLI, please indent them so they're under the bullet

> - **<Constant name="cloud_cli" />:** Supports the same linting [commands](/docs/cloud/configure-cloud-cli#lint-sql-files) as dbt Core:

Reviewer: Since line 85 was retained, we can probably remove the link here as they both point to the same page

> - **<Constant name="cloud_cli" />:** Supports the same linting [commands](/docs/cloud/configure-cloud-cli#lint-sql-files) as dbt Core:

Reviewer: ```suggestion
- **<Constant name="cloud_cli" />:** Supports the same linting [commands](/docs/cloud/configure-cloud-cli#lint-sql-files) as <Constant name="core" />:
```

    ```
    dbt sqlfluff lint [PATHS]... [flags]
    ```

    If no path is specified (for example, `dbt sqlfluff lint`), all SQL files in the project are linted.

>     See [Lint SQL files](/docs/cloud/configure-cloud-cli#lint-sql-files) for more information.

Author: ```suggestion

```
